### DISCUSSION TITLE
deploying

### DISCUSSION THREAD

**Discussion by lukehodges** [medusajs/medusa#1623] [Q&A] [answered]
any tips on how i could easily deploy medusa with all of its packages (ie redis and sqlite)
thanks alot for this

**Comment by olivermrbl** [comment, marked as answer]
In our [documentation](https://docs.medusajs.com), you'll find guides on deploying Medusa to:
- [Heroku](https://docs.medusajs.com/how-to/deploying-on-heroku)
- [DigitalOcean](https://docs.medusajs.com/how-to/deploying-on-digital-ocean)
- [Qovery](https://docs.medusajs.com/how-to/deploying-on-qovery)

Additionally, you likewise find guides for setting up Medusa Admin on Netlify.

Hope this helps!

**Comment by lukehodges** [comment]
cheers
